### DISCUSSION TITLE
排除特定域名

### DISCUSSION THREAD

**Discussion by caolib** [clash-verge-rev/clash-verge-rev#2482] [Q&A]
### 我想知道怎么排除指定域名，比如所有以cn为后缀的域名不走代理

**Comment by louiewh** [comment]
我也有这个需求，或者现在在软件哪里可以配置？

**Reply by louiewh** [reply]
在 ClashX 上是可以手工添加一些额外的规则的，有些网站特殊，无法通过代理访问。

**Comment by Arcticn** [comment]
可以参考下我的配置，额外单独增设自定义组别来实现全局的路由规则 https://github.com/clash-verge-rev/clash-verge-rev/discussions/2606
路由规则参考官方文档https://wiki.metacubex.one/config/rules/
